Assignment2/qpengcheng-pengcheng.ttl

```diff
@@ -0,0 +1,12 @@
+@base  <http://www.fi.upm.net/v1>.
+@prefix vocab: <http://www.fi.upm.net/v1#>.
+@prefix hasTemperature :<http://www.fi.upm.net/v1#hasTemperature>
+@prefix atTime : <http://www.fi.upm.net/v1#atTime>
+@prefix hasName: <http://www.fi.upm.net/v1#hasName>
+:Class01  		vocab:includes :Sensor029 ;
+				vocab:includes :Computer101.
+:Sensor029  	vocab:hasMeasurement :Measurement8401 ;
+:Computer101	vocab:hasOwner	:User10A;
+:Measurement8401 hasTemperature:integer 2;
+				 atTime:dateTime 2010-06-12T12:00:12.
```

Comment:
You have to add a datatype. Check how others have done it.

Reply:
done